lib/EvalSourceMapDevToolPlugin.js

```diff
@@ -154,7 +154,6 @@ class EvalSourceMapDevToolPlugin {
 						sourceMap.sources = moduleFilenames;
 						sourceMap.sourceRoot = options.sourceRoot || "";
 						const moduleId = chunkGraph.getModuleId(m);
-						sourceMap.file = `${moduleId}.js`;
```

Comment:
I think we should check here, in some cases we have the extension but with other no, can you improve this, thank you

Reply:
yep, of course, thanks for the feedback